demo-configs/test.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove these two extra files from your branch (`git rm demo-configs/test.json test_bash_script` then commit and push again)